docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -6,13 +6,13 @@ VTOL is a mod manager for installing and managing mods for the Northstar client
 
 ![VTOL Main Window](../../images/vtol-main-window.png)
 
-**The *Main Menu* of VTOL is where you'll probably spend most of your time.**
+__The _Main Menu_ of VTOL is where you'll probably spend most of your time.__
```

Comment:
Pretty sure you intended that line to still stay bold? I.e. it should be

```suggestion
**The _Main Menu_ of VTOL is where you'll probably spend most of your time.**
```

?